src/MusicStore/project.json

```diff
@@ -5,9 +5,6 @@
     "description": "Music store application on ASP.NET 5",
     "version": "1.0.0-*",
     "compilationOptions": { "warningsAsErrors": true, "define": [ "DEMO", "TESTING" ] },
-    "compile": [
-        "../../test/E2ETests/compiler/shared/**/*.cs"
-    ],
```

Comment:
I was referring to https://github.com/aspnet/MusicStore/blob/dev/test/E2ETests/PublishAndRunTests.cs